distributor-node/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### 1.2.0
 
+- Integrates OpenTelemetry API/SDK with Argus for exporting improved tracing logs & metrics to Elasticsearch. Adds `./start-elasticsearch-stack.sh` script to bootstrap elasticsearch services (Elasticsearch + Kibana + APM Server) with all the required configurations. 
```

Comment:
Since 1.2.0 has been already released and is in use, let's bump the version please